### DISCUSSION TITLE
Version 1.55.0 depends on git version v2.41.0 or newer

### DISCUSSION THREAD

**Discussion by kbck** [golangci/golangci-lint#4151] [Q&A] [answered]
The revgrep module (version 0.5.0) calls git with the argument "--default-prefix", this was added to git in version v2.41.0. 
On many platforms the version of git isn't the latest, and the above thus causes issues and the error message printed isn't that easy 

I can't seem to find documentation of minimum required version of git, is there such information?

**Comment by justinrush** [comment]
confirmed that if I add the edge version of git in alpine it works fine.

`apk add  --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/main git=2.42.0-r0`

**Comment by ldez** [comment, marked as answer]
The problem will be fixed by https://github.com/golangci/golangci-lint/pull/4154
